.github/workflows/integration_test_charm.yaml

```diff
@@ -168,6 +168,7 @@ jobs:
           sudo apt-get install python3-pip python3-venv -y
           python3 -m pip install --user pipx
           python3 -m pipx ensurepath
```

Comment:
Maybe `pipx ensurepath --global` if executed with necessary privileges. Do we have sudo there?